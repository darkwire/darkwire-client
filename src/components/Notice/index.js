import React from 'react';

export default props => (
  <div className="notice">
    {props.children}
  </div>);
