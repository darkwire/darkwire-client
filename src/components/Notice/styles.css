:local(.styles) {
  .info { 
    color: blue
  }
  .warning {
    color: yellow
  }

  .danger {
    color: gold
  }
}
