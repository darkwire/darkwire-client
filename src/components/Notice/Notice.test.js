import React from 'react'
import renderer from 'react-test-renderer'
import Notice from './index.js'

test('Notice Component', () => {
  const component = renderer.create(
    <Notice>
      <div>Hello world</div>
    </Notice>
  )
  const componentJSON = component.toJSON()

  expect(component).toMatchSnapshot()
  expect(componentJSON.children.length).toBe(1)
  expect(componentJSON.children[0].type).toBe('div')
  expect(componentJSON.children[0].children[0]).toContain('Hello world')
})
