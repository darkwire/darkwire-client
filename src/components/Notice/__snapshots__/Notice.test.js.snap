// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Notice Component 1`] = `
<div
  className="notice"
>
  <div>
    Hello world
  </div>
</div>
`;
